doc/schemas/listpays.schema.json

```diff
@@ -87,8 +87,16 @@
                 "bolt11": {},
                 "description": {},
                 "bolt12": {},
-                "amount_msat": {},
-                "amount_sent_msat": {},
+                "amount_msat": {
+                  "type": "msat",
+                  "description": "The amount of millisatoshi we intended to send to the destination",
+                  "deprecated": false
+                },
+                "amount_sent_msat": {
+                  "type": "msat",
+                  "description": "The amount of millisatoshi we sent in order to pay (may include fees and not match amount_msat)",
+                  "deprecated": false
+                },
```

Comment:
Hmm, deprecated wants a version now... not sure what it looks like without these fields...

Reply:
Yeah, I only added them because the diff used in `doc/Makefile` to detect removed fields that hadn't been marked as deprecated is having a heart attack because I dared to expand `{}` into a multiline dict (hence the field name line is removed.

Commenting out that diff temporarily and think about how we can improve this check.